tls/s2n_tls.h

```diff
@@ -26,12 +26,18 @@ extern int s2n_flush(struct s2n_connection *conn, s2n_blocked_status * more);
 extern int s2n_client_hello_send(struct s2n_connection *conn);
 extern int s2n_client_hello_recv(struct s2n_connection *conn);
 extern int s2n_sslv2_client_hello_recv(struct s2n_connection *conn);
+extern int s2n_client_hello_retry_send(struct s2n_connection *conn);
```

Comment:
Is this intended to be a handler for a `HelloRetryRequest` message or the retried `ClientHello` sent in response to `HelloRetryRequest`?

Reply:
For the HelloRetryRequest. I was working under the assumption that the resent ClientHello would use the same handlers as the original ClientHello, but that might not be true. We may need more handler stubs :)